### ISSUE TITLE
Bastok 7-1 not giving key item after golem fight.

### ISSUE THREAD

**Issue by gweivyth** [EdenServer/community#754]
As the title suggests, after fighting the golems for 7-1, I was unable to get the key item from the ???.  Godmode helped me get past this mission by manually giving me the key item, this bug report is mostly here to remind someone to look at this mission at a later date. 

Additionally, the ??? does not appear to move around in 15 minute intervals as it should.  We sat at the same ??? for 30+ minutes and it never moved, although the ??? did move while we were fighting the golems.  Could have been part of the problem.

Here's a link to the mission description.

https://www.bg-wiki.com/bg/Bastok_Mission_7-1

**Comment by panfunk** [comment]
Just had the same issue, killed both Golems, searched for ??? again, but didn't get anything when clicking it.
Chars: Pan & Chan

**Comment by panfunk** [comment]
Okay I was able to turn in the Mission at Cid. I checked my KIs before leaving Ro'Maeve and I'm 99,9% sure I didn't have the necessary KI or got he msg that I attained it, but Cid still was grateful I got him the 'reinforced cermet'

![pan_2019 01 18_154305](https://user-images.githubusercontent.com/36666121/51393436-fd664500-1b37-11e9-8279-e08a3d7f6255.png)

**Comment by Volkaru** [comment]
Can confirm this is still an issue. Clicked the second ???. Got no message of a Key Item, or a KI being in my list. But Cid advanced the mission.

**Comment by Mikokiko** [comment]
Can confirm, did not get KI after looking for the ??, but was able to advance

**Comment by AnkleAssassinz** [comment]
08/20/2019: Just confirming the issue is still happening where clicking the ??? after killing the 2 golem NM does not give KI but talking to Cid still lets you complete the quest line.

-AnkleAssassin

**Comment by Camuel666** [comment]
so i did this fight and the ??? never reappeared. I ended up dying and Hp'd when i went back i found the ??? and clicked on them and was ready to fight the golems again but no golems spawned. I clicked on it again and it told me nothing out of the ordinary was there. Went to cid and was able to progress through the mission but i did not get a message saying i obtained the KI nor was the KI in any of my KI inventories. I believe this is still bugged as when zoning iwithout getting the KI i should have had to fight the golems again
